application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -35,7 +35,7 @@
 /**
  * Implements the /modmail command, which allows users to contact a moderator within the server
  * which forwards messages to moderators in a dedicated channel given by
- * {@link Config#getModAuditLogChannelPattern()}.
+ * {@link Config#getModMailChannelPattern()}.
```

Comment:
change the Config.java and config template too

Reply:
it was properly written in the config just a mistake in modmail comment